feature/bgp/multipath/otg_tests/bgp_multipath_ecmp_test/README.md

```diff
@@ -0,0 +1,58 @@
+# RT-1.51: BGP multipath ECMP
+
+## Summary
+
+Validate BGP in multipath scenario
+
+## Testbed type
+
+[TESTBED_DUT_ATE_4LINKS](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+*  Connect DUT port-1 and 2 to ATE port-1 and 2 respectively
+*  Configure IPv4/IPv6 addresses on the interfaces
+
+*  RT-1.51.1: Verify eBGP - multipath not enabled
+  *  Establish eBGP sessions between:
+    *  ATE port-1 and DUT port-1
+    *  ATE port-2 and DUT port-2
+    *  ATE port-3 and DUT port-3
+  *  Configure ATE devices(ports) on same AS
+  *  Enable an Accept-route all import-policy/export-policy for eBGP session under the neighbor AFI/SAFI.
+  *  Validate: Advertise same prefixes on both sessions from ATE ports, observe received prefixes at DUT ports.
+
+*  RT-1.51.2: Enforcing multipath hence ECMP scope to only one peer AS
+  *  Configure ATE devices(ports) on same AS and connected to one internal target network
+  *  Enable multipath and set maximum-paths limit to 2
+  *  Advertise equal cost paths from 3 interfaces of ATE of same AS
+  *  Validate: heck entries in FIB for advertised prefix, it should only have 2 entries
+
```

Comment:
Also add check by sending traffic.  Since in 1.51.1 we verified maximum-paths = 1 (the default),  then this test can just maximum-paths = 2.